packages/default/scss/menu/_index.scss

```diff
@@ -2,6 +2,17 @@
 @import "../utils/_index.scss";
 
 
+// Module meta
+$_kendo-module-meta: (
+    name: "menu-button",
```

Comment:
- [ ] name should be menu
- [ ] change dependencies with the correct ones for the menu